cmd/nerdctl/compose/compose_ps.go

```diff
@@ -44,7 +45,7 @@ func newComposePsCommand() *cobra.Command {
 	return composePsCommand
 }
 
-type composeContainerPrintable struct {
+type ContainerPrintable struct {
```

Comment:
nit: not sure if the renaming is necessary. In the future if we want to refactor and move these `struct` to a common pkg like `api/types`, this might conflict with the one used in `nerdctl ps`.